identifiers/country-au/federal_electorates.csv

```diff
@@ -1,151 +1,162 @@
+ocd-division/country:au/state:nsw/federal_electorate:charlton,New South Wales - Federal House of Representatives Charlton electorate,,2016
+ocd-division/country:au/state:nsw/federal_electorate:chifley,New South Wales - Federal House of Representatives Chifley electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:cook,New South Wales - Federal House of Representatives Cook electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:cowper,New South Wales - Federal House of Representatives Cowper electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:cunningham,New South Wales - Federal House of Representatives Cunningham electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:dobell,New South Wales - Federal House of Representatives Dobell electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:eden-monaro,New South Wales - Federal House of Representatives Eden-Monaro electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:farrer,New South Wales - Federal House of Representatives Farrer electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:fowler,New South Wales - Federal House of Representatives Fowler electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:gilmore,New South Wales - Federal House of Representatives Gilmore electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:grayndler,New South Wales - Federal House of Representatives Grayndler electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:greenway,New South Wales - Federal House of Representatives Greenway electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:hughes,New South Wales - Federal House of Representatives Hughes electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:hume,New South Wales - Federal House of Representatives Hume electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:hunter,New South Wales - Federal House of Representatives Hunter electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:kingsford_smith,New South Wales - Federal House of Representatives Kingsford Smith electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:lindsay,New South Wales - Federal House of Representatives Lindsay electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:lyne,New South Wales - Federal House of Representatives Lyne electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:macarthur,New South Wales - Federal House of Representatives Macarthur electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:mackellar,New South Wales - Federal House of Representatives Mackellar electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:macquarie,New South Wales - Federal House of Representatives Macquarie electorate,,
+ocd-division/country:au/state:nsw/federal_electorate:mcmahon,New South Wales - Federal House of Representatives McMahon electorate,2010,

```

Comment:
I see some validFrom dates added for 2009 and 2010 electorates. Is there a specific reason to do this? I think it'd be reasonable to either have the validFrom dates added for districts added after this file was first created, or for all districts. The latter would be a great deal of work with little benefit, so  my preference would be for the former.

Reply:
Done